core/src/com/unciv/models/ModConstants.kt

```diff
@@ -109,6 +109,9 @@ class ModConstants {
     var tributeGlobalModifier = 100 // 75 in BNW
     var tributeLocalModifier = 100 // 125 in BNW
 
+    // Forest chops: if Production bonus from removing terrain features should increase with game progress
+    var choppingYieldsIncreaseWithGameProgress = false // true in Civ6
```

Comment:
This is something that should be a unique modifier, like "(modified by game speed)" is
Not a global configuration
That way it
- Can be applied selectively
- Can be applied everywhere that 'modified by game speed' can be applied

The 100% -> 1000% should also be part of the unique, like "<increasing with game progress to [1000]%>"

Reply:
If it is to be a new metamodifier unique, if makes sense to perform the calculation for this separarely from tryProvideProductionToClosestCity. In which file should I put the new modifyByGameProgress function?

Reply:
Damn, that's a good question... I don't have a good answer for it 0_o

Reply:
I locally put it in Unique.kt as it seemed most appropriate, but some reason the ModifiedByGameProgress unique is not recognized, unlike the ModifiedByGameSpeed unique which I used as a template.